NCS.DSS.GDPRUpdateCustomerInformation/Function/GDPRUtility.cs

```diff
@@ -1,26 +1,32 @@
-using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Azure.Functions.Worker;
 using Microsoft.Extensions.Logging;
-using NCS.DSS.GDPRUpdateCustomerInformation.Services;
+using NCS.DSS.GDPRUtility.Services;
 
-namespace NCS.DSS.GDPRUpdateCustomerInformation.Function
+namespace NCS.DSS.GDPRUtility.Function
 {
-    public class GDPRUpdateCustomerInformation
+    public class GDPRUtility
     {
         private readonly IIdentifyAndAnonymiseDataService _identifyAndAnonymiseDataService;
-        private readonly ILogger<GDPRUpdateCustomerInformation> _logger;
+        private readonly ILogger<GDPRUtility> _logger;
 
-        public GDPRUpdateCustomerInformation(IIdentifyAndAnonymiseDataService identifyAndAnonymiseDataService, ILogger<GDPRUpdateCustomerInformation> logger)
+        public GDPRUtility(IIdentifyAndAnonymiseDataService identifyAndAnonymiseDataService, ILogger<GDPRUtility> logger)
         {
             _identifyAndAnonymiseDataService = identifyAndAnonymiseDataService;
             _logger = logger;
         }
 
-        [Function(nameof(GDPRUpdateCustomerInformation))]
-        public async Task<IActionResult> RunAsync([HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = null)] HttpRequest req)
+        /*
+        In case of failed execution, FixedDelayRetry will retry up to three times after a 15 second delay interval.
```

Comment:
Thanks for adding - can you please add a link to this page: https://crontab.cronhub.io/ as it is a NCRONTAB calculator

Reply:
![image](https://github.com/user-attachments/assets/4cc9a935-870c-4e00-b2e8-3185678ca90f)